CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 ## unreleased
 
+### Changed
+
+- Attribute value for `sdk.queries.fileevents.filters.file_filter.FileCategory.*` changed to "CamelCase" format.
```

Comment:
I don't think this is needed since it is more of an implementation detail

Reply:
Removed.